platforms/hyperledger-besu/configuration/README.md

```diff
@@ -1,49 +1,49 @@
+There are two approaches to deploy a DLT network using BAF: 
+- Use a dedicated instance to deploy and manage the DLT network. This is recommended for production environments, and requires some setup & manual installation of CLI tools. More info on setting up this dedicated instance is found [here](https://blockchain-automation-framework.readthedocs.io/en/latest/operations/configure_prerequisites.html#ansible-inventory-file).
+- Use the 'BAF Build container' to create a containerized Ansible controller from which to deploy/manage your networks. This is recommended for development instances, as it is an easy way to build the required base environment for BAF deployment. More info can be found [here](https://blockchain-automation-framework.readthedocs.io/en/latest/developer/docker-build.html).
 
-## Prerequisites
+## Installation pre-requisites
+Some tools are required to run the BAF Automation, such as Kubernetes, Git (repository), Vault and more.
+For more information on the installation pre-requisites, please refer to [this guide](https://blockchain-automation-framework.readthedocs.io/en/latest/prerequisites.html).
 
-To run the playbooks, following are the pre-requisites.
-1. Ansible 2.8.2 with jmespath installed (a docker image can be made from this [Dockerfile](../../shared/images/ansibleSlave.Dockerfile)), or use the dockerfile given in the root.
-2. Ansible controller configured like this [sample](../../shared/inventory) inventory settings.
-3. One Managed Kubernetes cluster for each organization (currently tested on EKS so you need **AWS cli** credentials).
-4. A Hashicorp Vault installation for each organization which is initiated and unsealed. The Vault Address should be accessible from this machine (where the playbook is run) and the Kubernetes cluster, and also, the Vault root token is available.
-5. A Git User with write access to all the branches in the chosen Git repository. This user should be a service user and not a federated user.
-6. Edited and saved network.yaml with all fields populated according to your requirements. A Sample network.yaml can be found [here](./samples/network-besu.yaml)
+## Configuration pre-requisites
+For each organization in the DLT network you need to set up the following:
+1. One Managed Kubernetes cluster; BAF is currently tested on Amazon EKS, which means you will need AWS CLI set up as well.
+2. A Hashicorp Vault installation for each organization which is initialized and unsealed. The Vault Address should be accessible from this machine (where the playbook is run) and the Kubernetes cluster. The Vault root token is used in the network configuration, so this should be available as well.
+3. A Git User with write access to all the branches in the chosen Git repository; as well as an access token.
+4. The network configuration file (`network.yaml`) which has been filled in according to your requirements. A sample `network.yaml` for Hyperledger Besu can be found [here](./samples/network-besu.yaml).
```

Comment:
"Sample networkyamls ..." and link to the samples directory only, not a specific file

Reply:
Resolved in commit [9f27d09](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/9f27d09fc1063e42936f0cdc81410dc63f7106e9)